mhs-reference-implementation/mhs/SDSLookup/tests/test_sds_handler.py

```diff
@@ -0,0 +1,36 @@
+from unittest import TestCase
+import mhs.SDSLookup.sds_handler as sds
+
+import mhs.SDSLookup.tests.ldap_mocks as mocks
+from utilities.test_utilities import async_test
+
+NHS_SERVICES_BASE = "ou=services, o=nhs"
+
+MHS_OBJECT_CLASS = "nhsMhs"
+
+OPENTEST_SDS_URL = "192.168.128.11"
+PARTY_KEY = " A91461-9199084"
```

Comment:
These look like your actual party key - probably worth anonymising these values.

Reply:
To my knowledge these are useless without my vpn creds alongside this, so this shouldn't be a concern. I've changed them to be fake values and replaced the data in the LDAP test data json's so the tests still work without opentest connection. I'll be more careful with what creds I'm committing next time!

Reply:
Yes, I don't think it's the end of the world, but probably good to avoid doing.